ImageLounge/src/main.cpp

```diff
@@ -320,6 +320,9 @@ int main(int argc, char *argv[])
         cw->startSlideshow();
     }
 
+    if (cw->hasViewPort())
+        ((QWidget *)cw->getViewPort())->setFocus(Qt::TabFocusReason);
```

Comment:
You can use here

`cw->getViewPort()->setFocus(Qt::TabFocusReason);`

when you add

`#include "DkViewPort.h"`

Reply:
My thought at the time was to avoid the dependency as it isn't needed. The worst case is the cast becomes invalid due to refactoring and crashes the app at startup; not in a place you'd have to uncover. But if you prefer the other way, no problem.